test/CMakeLists.txt

```diff
@@ -760,41 +760,6 @@ if(${MIOPEN_TEST_WITH_MIOPENDRIVER})
     )
 endif()
 
-# ./bin/MIOpenDriver conv -n 128 -c 1024 -H 14 -W 14 -k 2048 -y 1 -x 1 -p 0 -q 0 -u 2 -v 2 -l 1 -j 1 -m conv -g 1 -F 1 -t 1
-# MIOPEN_DEBUG_CONV_IMMED_FALLBACK=0
-if(MIOPEN_EMBED_DB)
-    set(MIOPEN_EMBED_TEST_ARG ${MIOPEN_TEST_FLOAT_ARG} --disable-validation --verbose)
-    # WORKAROUND for issue #874
-    set(MIOPEN_WA_ISSUE_874_F  MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0)
-    set(MIOPEN_WA_ISSUE_874_W  MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_WRW_V4R1=0)
-    set(MIOPEN_WA_ISSUE_874_FW MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0 MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_WRW_V4R1=0)
-    # WORKAROUND for issue #1008
-    set(MIOPEN_WA_ISSUE_1008 MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2=0)
```

Comment:
@alexandraBara We don't need these workarounds anymore?

Reply:
@alexandraBara Maybe we really do not need these. Let me check.

Reply:
> @alexandraBara We don't need these workarounds anymore?

@atamazov , confimed with @shurale-nkn , still need these workarounds for now

Reply:
@xinlipn #2179 is merged in

Reply:
> @xinlipn #2179 is merged in

@atamazov Could you resolve this conversation if this has been resolved?

Reply:
[Resolved]

Note that I do no have enough permissions to mark comment thread as "resolved".

Reply:
[Notice] W/As for #874 and #1008 finally removed by #2220